src/agent/research_assistant.py

```diff
@@ -31,10 +31,9 @@ class AgentState(MessagesState, total=False):
 
 # NOTE: models with streaming=True will send tokens as they are generated
 # if the /stream endpoint is called with stream_tokens=True (the default)
-models = {
-    "gpt-4o-mini": ChatOpenAI(model="gpt-4o-mini", temperature=0.5, streaming=True),
-}
-
+models: dict[str, str] = {}
```

Comment:
wouldn't it be `dict[str, Any]` (`langchain.BaseChatModel` or whatever it is)

Reply:
fixed